pkg/sink/interceptors.go

```diff
@@ -42,27 +42,10 @@ func GetURI(objRef *corev1.ObjectReference, ns string) (*url.URL, error) {
 	return nil, xerrors.New("Invalid objRef")
 }
 
-// TODO: Use request.Clone once we move to go 1.13
 func createOutgoingRequest(ctx context.Context, original *http.Request, url *url.URL, payload []byte) *http.Request {
-	r := original.WithContext(ctx)
-	// RequestURI cannot be set in outgoing requests
-	r.RequestURI = ""
+	r := original.Clone(ctx)
+	//r.RequestURI = "" // RequestURI cannot be set in outgoing requests
```

Comment:
Delete this commented out code?

Reply:
Hmmm, I think we actually need that...Why didn't a test fail? :thinking:

Reply:
Fixed: https://github.com/tektoncd/triggers/pull/208/files#r343850505